source/common/http/http3/conn_pool.cc

```diff
@@ -83,7 +83,7 @@ Http3ConnPoolImpl::Http3ConnPoolImpl(
     CreateClientFn client_fn, CreateCodecFn codec_fn, std::vector<Http::Protocol> protocol,
     OptRef<PoolConnectResultCallback> connect_callback, Http::PersistentQuicInfo& quic_info)
     : FixedHttpConnPoolImpl(host, priority, dispatcher, options, transport_socket_options,
-                            random_generator, state, client_fn, codec_fn, protocol),
+                            random_generator, state, client_fn, codec_fn, protocol, {}, nullptr),
```

Comment:
Should we consider doing this for the h3 pool too?

Reply:
probably?  I thought HTTP/3 had a different way of opening stream window and I thought it might be more reactive to server load than HTTP/2 settings tend to be but probably no one actually does that :-P

Reply:
Yeah, the mechanism for allowing a new stream to be opened is implicit in HTTP/2 (you can open once you close) whereas it's explicit in HTTP/3 (even if you close a stream you still need to wait until the peer says you can open a new one). That being said, during the QUIC handshake, there is an "initial_max_streams" parameter which is sent that is the initial limit. It's basically analogous to this value in HTTP/2. No need to do address this in the current PR, but it's probably worth a TODO or an issue?